metashare/api/views.py

```diff
@@ -45,9 +45,7 @@ class CreatePrMixin:
     def create_pr(self, request, pk=None):
         serializer = CreatePrSerializer(data=self.request.data)
         if not serializer.is_valid():
-            return Response(
-                serializer.errors, status=status.HTTP_422_UNPROCESSABLE_ENTITY
-            )
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
@wlonk As I was testing form errors, I realized in this project it looks like we've been inconsistent between `422` and `400`. In another project we decided to standardize on `400` -- are you okay doing that here too? I'm also okay adjusting the front end so that it considers either `400` or `422` to be a candidate for inline form errors.

Reply:
Yeah, I picked up on those changes reading through it, and I like them.